cmd/infracost/testdata/breakdown_terraform_sync_usage_file/breakdown_terraform_sync_usage_file.golden

```diff
@@ -8,7 +8,7 @@ Project: infracost/infracost/cmd/infracost/testdata/breakdown_terraform_sync_usa
  └─ Insights queries data scanned                    Monthly cost depends on usage: $0.005 per GB   
                                                                                                     
  aws_cloudwatch_log_group.production_logs["media"]                                                  
- ├─ Data ingested                                    Monthly cost depends on usage: $0.50 per GB    
+ ├─ Data ingested                                             1,000  GB                     $500.00 
```

Comment:
This seems like this was a bug before that was fixed by this PR, the resource defaults are as follows:

```
resource_type_default_usage:
  aws_cloudwatch_log_group:
    monthly_data_ingested_gb: 1000.0
```